src/app/server-cluster/testing/EmptyProvider.h

```diff
@@ -0,0 +1,62 @@
+/*
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <app/data-model-provider/Provider.h>
+#include <protocols/interaction_model/StatusCode.h>
+
+namespace chip {
+namespace Test {
+
+/// A provider that is emtpy - everything returns null
```

Comment:
No, some of that stuff returns empty lists, errors, etc....  Needs a clearer description.